### PR TITLE
Character Movement

### PR DESCRIPTION
**Pull request by shulfo**
Setup character controller with massive help from Liquid.

### CODE REVIEW

**Review by Beelzenord — DISMISSED** [review]

File: WalkingHorror.uproject [hunk 1]
```diff
@@ -1,13 +1,16 @@
 {
 	"FileVersion": 3,
-	"EngineAssociation": "5.4",
+	"EngineAssociation": "5.3",
```
Comment (mychallipson) [inline WalkingHorror.uproject]: Did we intend to bump this back down to 5.3?

**Comment by mychallipson** [comment]
I also humbly request that we invert the y axis in the player control

File: WalkingHorror.uproject (hunk 1 above)
Comment (Beelzenord) [reply WalkingHorror.uproject]: We thought it was causing issues importing dependencies for 'enhancedinput' header files. The Enhanced Input library is UE 5.+ industry standards for handling user inputs. The way it was done before is now deprecated. Turns out we didn't tell the Build.cs files that we needed it. We bumped it down to see if we could import the header files. The superstition was unrewarded. We wanted to bump it to 5.4 to exploit the easy rigging for meshes.

Comment (shulfo) [reply WalkingHorror.uproject]: Good catch. I think we decided to bump it back up to 5.4 after we confirmed the problem was most likely unrelated to changing it to 5.3. I should be able to update this branch with Liquid's most recent commit and all will be right again.

**Review by Beelzenord — APPROVED** [review]

**Review by mrshaiguy — APPROVED** [review]

**Pull request merged**